webapp/home/subdomains/proteomics/content/dda_tmt.py

```diff
@@ -0,0 +1,83 @@
+"""Content for data section - tools, workflows and help tabs."""
+
+galaxy_au_support_item = {
+    "title_html": 'Galaxy Australia support',
+    "description_html": """<p>
+            Any user of Galaxy Australia can request support through an
+            online form.
+        </p>""",
+    "button_link": "/request/support",
+    "button_html": "Request support",
+}
+
+tools = [
+    # {   # Accordion item schema:
+    #     "title_html": '',
+    #     "description_html": """""",
+    #     "inputs": [
+    #         {
+    #             'datatypes': [''],
+    #             'label': '',
+    #         },
+    #     ],
+    #     "button_link": "",
+    #     "button_html": "",
+    #     "button_tip": "",
+    #     "view_link": "",
+    #     "view_html": "",
+    #     "view_tip": "",
+    # },
+    {
+        "title_html": '<code>MaxQuant</code>',
+        "description_html": '<p>MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.</p>',
+        "inputs": [
+            {'datatypes': ['thermo.raw']},
+            {'datatypes': ['mzML']},
+            {'datatypes': ['mzXML']},
```

Comment:
If these three datatypes are the MS spectra input file, should be:
```py
{
    'datatypes': [
        'thermo.raw',
        'mzML',
        'mzXML',
    ],
    'label': 'MS spectra (input file)',
},
```
Otherwise this implies there are three separate input files!